### PR TITLE
Added Notification Templates entity

### PR DESCRIPTION
**Pull request by rbatistadev**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Added Notification Template Entity to platform portal

### CODE REVIEW

File: web/portal/platform/src/entities/Language/Language.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  },
+};
+
+const Language: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: AccountTreeIcon,
+  iden: 'Language',
+  title: _('Language', { count: 2 }),
+  path: '/languages',
+  toStr: (row: LanguagePropertyList<EntityValues>) =>
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Language/Language.tsx:31]: `import { getI18n } from 'react-i18next';

toStr: (row: LanguagePropertyList<EntityValues>) => {
    const language = getI18n().language.substring(0, 2);
    const name = row.name as Record<string, string>;
    return name[language];
  },`

File: web/portal/platform/src/entities/Language/SelectOptions.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+const LanguageSelectOptions: SelectOptionsType = ({
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const Language = entities.Language;
+
+  return defaultEntityBehavior.fetchFks(
+    Language.path,
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Language/SelectOptions.ts:14]: order by name

File: web/portal/platform/src/entities/NotificationTemplateContent/Form.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  const fkChoices = useFkChoices({
+    foreignKeyGetter,
+    entityService,
+    row,
+    match,
+  });
+
+  const groups: Array<FieldsetGroups | false> = [
+    {
+      legend: _('Main'),
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplateContent/Form.tsx:21]: there are two sections with different labels

File: web/portal/platform/src/entities/Language/ForeignKeyResolver.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { autoForeignKeyResolver } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { foreignKeyResolverType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { LanguagePropertiesList } from './LanguageProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Language/ForeignKeyResolver.tsx:5]: remove this file

File: web/portal/platform/src/entities/NotificationTemplate/ForeignKeyGetter.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { NotificationTemplatePropertyList } from './NotificationTemplateProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/ForeignKeyGetter.tsx:5]: remove this file

File: web/portal/platform/src/entities/NotificationTemplate/ForeignKeyResolver.tsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import { autoForeignKeyResolver } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { foreignKeyResolverType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { NotificationTemplatePropertiesList } from './NotificationTemplateProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/ForeignKeyResolver.tsx:5]: remove this file

File: web/portal/platform/src/entities/Language/Language.tsx (hunk 1 above)
Comment (rbatistadev) [reply web/portal/platform/src/entities/Language/Language.tsx:31]: Thank you so much, for the tip, working on it.

File: web/portal/platform/src/entities/NotificationTemplateContent/SelectOptions.ts [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const InvoiceTemplate = entities.InvoiceTemplate;
+
+  return defaultEntityBehavior.fetchFks(
+    InvoiceTemplate.path,
+    ['id', 'name'],
+    (data: Array<EntityValues>) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplateContent/SelectOptions.ts:16]:
```suggestion
    (data: NotificationTemplateContentPropertiesList) => {
```

**Review by fmendozzac — APPROVED** [review]

File: web/portal/platform/src/entities/NotificationTemplateContent/ForeignKeyGetter.tsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { NotificationTemplateContentPropertyList } from './NotificationTemplateContentProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/NotificationTemplateContent/ForeignKeyGetter.tsx:5]: Remove this file

File: web/portal/platform/src/entities/NotificationTemplateContent/ForeignKeyResolver.tsx [hunk 9]
```diff
@@ -0,0 +1,22 @@
+import { autoForeignKeyResolver } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { foreignKeyResolverType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { NotificationTemplateContentPropertiesList } from './NotificationTemplateContentProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/NotificationTemplateContent/ForeignKeyResolver.tsx:5]: Remove this file

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Pull request merged**